### PR TITLE
VAG-74: Created clients byID GET

### PR DESCRIPTION
**Pull request by Vladosik12333**

### CODE REVIEW

File: src/controllers/clients.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+const { createClientService, getByIDService } = require('../services/clients');
 
 const createClient = async (req, res) => {
 	const client = await createClientService(req.body);
 
 	res.status(201).json({ client });
 };
 
-module.exports = { createClient };
+const getByID = async (req, res) => {
```
Comment (gnatuk2312) [inline src/controllers/clients.js]: Please rename getByID into `getClientByID`

File: src/services/clients.js [hunk 2]
```diff
@@ -6,4 +6,10 @@ const createClientService = async (body) => {
 	return client;
 };
 
-module.exports = { createClientService };
+const getByIDService = async (id) => {
```
Comment (gnatuk2312) [inline src/services/clients.js]: Please rename into `getClientByIDService`

**Review by gnatuk2312 — CHANGES_REQUESTED** [review]

File: src/routes/clients.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
 const router = require('express').Router();
 const Joi = require('joi');
+Joi.objectId = require('joi-objectid')(Joi);
 const wrapper = require('../middleware/wrapper');
 const validator = require('../middleware/validator');
 
-const { createClient } = require('../controllers/clients');
+const { getByID, createClient } = require('../controllers/clients');
+
+const validatorGetById = Joi.object({
```
Comment (gnatuk2312) [inline src/routes/clients.js:9]: Can we call this validator `validatorClientID` and use it for PUT and DELETE endpoints in future?

**Review by gnatuk2312 — CHANGES_REQUESTED** [review]

File: src/routes/clients.js (hunk 3 above)
Comment (Vladosik12333) [reply src/routes/clients.js:9]: We can if u rename it to "validatorClientID" before using it in the case when u need it.

**Review by gnatuk2312 — APPROVED** [review]

**Pull request merged**